ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/statementofmeans/ChildMapper.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.ccd.mapper.statementofmeans;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDChild;
+import uk.gov.hmcts.cmc.ccd.mapper.Mapper;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Child;
+
+@Component
+public class ChildMapper implements Mapper<CCDChild, Child> {
+
+    @Override
+    public CCDChild to(Child child) {
+        if (child == null) {
```

Comment:
Is that needed? Sometimes we have it, sometimes we don't. I would not expect to have it as in healthy app we should not attempt to map null child / have nulls in collections.